packages/workspace/src/tasks-runner/task-orchestrator.ts

```diff
@@ -339,8 +339,10 @@ export class TaskOrchestrator {
     const envsFromFiles = {
       ...parseEnv('.env'),
       ...parseEnv('.local.env'),
+      ...parseEnv('.env.local'),
```

Comment:
Sorry, you were totally right in your *Alternative Solution*.

We should have the existing `.local.env` take prescedent over the new `.env.local` in case users already have both. This should be closer to same behavior we had before. I made the change for you because for some reason CircleCI wasn't picking up your PR so I rebased it.